consensus/istanbul/core/commit_test.go

```diff
@@ -266,6 +266,243 @@ OUTER:
 	}
 }
 
+func TestHandleCommitV2(t *testing.T) {
```

Comment:
If we put all the v2 tests in their own file then removing the original tests becomes as simple as deleting a file, it's also going to make any conflicts easier to resolve, in the case that changes were made to the tests.

Reply:
And it would make reviewing a lot easier because then I could just diff the two files.

Reply:
Ok I've made this change in the below PR:
* #1954

Reply:
Merged